frontend/src/app/event/event-page/event-page.component.html

```diff
@@ -0,0 +1,55 @@
+<div>
+
+    <div class="parameter-bar">
+        <search-bar class="search-bar" [searchBarQuery]="searchBarQuery" (searchBarQueryChange)="onSearchBarQueryChange($event)"/>
+    </div>
+
+    <div class="container">
+        <div class="events-grid">
+
+            <div class="events-column">
+                <mat-card class="events-section" appearance="outlined" style="padding: 16px;">
+
+                    <div *ngFor="let date of eventsPerDay.keys()" style="padding-bottom: 16px;">
+                        <mat-card-title>{{date}}</mat-card-title>
+                        <mat-divider style="padding-bottom: 8px;" />  
+                        <event-card *ngFor="let event of eventsPerDay.get(date)" [event]="event"
+                        [disableLink]="innerWidth > 900" [selected]="event === selectedEvent"
+                        (onClicked)="onEventCardClicked($event)"
+                        />  
+                    </div>
```

Comment:
I believe this is the section causing the change after checked and a more idiomatic way to loop over key/value in map is: https://angular.io/api/common/KeyValuePipe

Reply:
This worked!!!